internal/pkg/template/template.go

```diff
@@ -59,23 +84,36 @@ func (t *Template) Read(path string) (*Content, error) {
 
 // Parse parses the template under "/templates/{path}" with the specified data object and returns its content.
 func (t *Template) Parse(path string, data interface{}, options ...ParseOption) (*Content, error) {
-	content, err := t.read(path)
+	tpl, err := t.parseTemplate("template", path, options...)
```

Comment:
Why is the `param` hardcoded to "template" here? I don't see it having much effect through the chain, but is this just necessary for the call to template.New()?

Reply:
> but is this just necessary for the call to template.New()?

Correct! The reason why they require a name I believe is for the scenario below with `ParseAppTemplate`.  
To illustrate, let's say I created a new Template object named "base" with:
```go
tpl := template.New("base").Parse(content)
```
where the content contains two nested templates "hello" and "world".
```go
{{define "hello"}}Hello, World!{{end}}
{{define "bye"}}Bye, World!{{end}}
```
Then I can decide whether to execute the entire template with `tpl.Execute(buf, data)` or I can specify a particular nested template with `tpl.ExecuteTemplate(buf, "hello", data)` that will render only "Hello, World!".

So yeah, if your named template won't be used anywhere else it's useless. But if you need to include it into another template then the name becomes important.